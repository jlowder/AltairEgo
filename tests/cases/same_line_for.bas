10 FOR I=1 TO 5: PRINT I: NEXT I
20 PRINT "Done"
