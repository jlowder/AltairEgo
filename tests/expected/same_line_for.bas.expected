Altair Ego: Emulating Altair BASIC 32K Rev. 3.2
OK
OK
 1
 2
 3
 4
 5
Done
OK
